src/standard_library/cyptographic_primitives.md

```diff
@@ -0,0 +1,103 @@
+
+# Cryptographic primitives
+
+Some cryptographic primitives are already developed and ready-to-use for any Noir project:
+
+## sha256
+
+Given an array of bytes, returns the sha256 of it:
+
+```rust
+// x = [72, 101, 108, 108, 111, 44, 32, 119, 111, 114, 108, 100, 33] (Hello, world!)
```

Comment:
Encoding "Hello, world!" as the input bytes here confused me a bit while reading it as I expected "Hello, world!" to be the output of println at first.

Reply:
Yeah I just put it there to shut up the compiler, then copied it without removing them 🤦‍♂️